docs/tutorials/on_saved.md

```diff
@@ -0,0 +1,116 @@
+# Benchmark on saved models
+There is a high likelihood that you, at some point, find yourself wanting to benchmark models that were trained previously.
+In this guide we will walk through how we can accomplish this with nnbench.
+We will start with an aside that talks through the setup of the example we will use in this guide.
+If you are only interested in the application of nnbench, you can skip this section.
```

Comment:
I would put these into the NER section below, and leave out the exposition (L4) or maybe reduce it to "This is a very short primer on the experiment setup".